Makefile

```diff
@@ -147,3 +147,7 @@ mock: $(MOCKGEN)
 
 clean-release:
 	rm -rf $(RELEASE_DIR)
+
+release: release-manifests
```

Comment:
@zulfilee please use this to generate release artifacts 

this targets builds release manifests in _build/release/ dir and upload docker images